cupy_backends/cuda/api/runtime.pxd

```diff
@@ -10,6 +10,7 @@ cdef class PointerAttributes:
         public int device
         public intptr_t devicePointer
         public intptr_t hostPointer
```

Comment:
I wonder why these members are marked "public" instead of "readonly"...Can I change them in this PR?

Reply:
i think its fine to change them here, thanks!

Reply:
Done in 2003438.